python/dlisio/plumbing/utils.py

```diff
@@ -343,8 +343,12 @@ def describe_text(buf, text, width, indent, subindent=None):
     if subindent is None: subindent = indent
 
     if text is None: text = str(None)
+    else: text = str(text)
+
+    if len(text) == 0: text = '""'
```

Comment:
1) yes, when no content the list is empty. However, the __repr__ of an empty list is `[]`, while an empty string is just an empty string. Given an attribute `attribute` where the value is an empty dict, the output in describe (for that line) becomes:` attribute : []`, but if the value was string, and an empty one, this would output as a blank line. This commit makes an empty string behave like an empty list and the line now becomes `attribute : ""`
2) it only applies if you explicitly tell describe to not exclude empty attributes